app/templates/user.html

```diff
@@ -29,4 +29,44 @@ <h1>User: {{ user.username }}</h1>
     {% if next_url %}
     <a href="{{ next_url}}">Older Posts</a>
     {% endif %}
+
+    <script>
+    function startFollowing() {
+        $.ajax({
+            url: '{{ url_for('core.follow', username=user.username)}}',
+            statusCode: {
+                200: function(data) {
+                    let btn = document.getElementById('follow');
+                    btn.onclick = stopFollowing;
+                    btn.id = 'unfollow';
+                    btn.innerText = 'Unfollow';
+                    console.log('gotem');
```

Comment:
please increase/decrease the follower count if the request is successful, modify the view functions accordingly.

Reply:
can you also show flash messages when user follows/unfollows the other user.